auto_install.sh

```diff
@@ -7,12 +7,12 @@ RPI_SETUP_REPO=vocalfusion-rpi-setup
 RPI_SETUP_DIR=$SETUP_DIR/$RPI_SETUP_REPO
 RPI_SETUP_SCRIPT=$RPI_SETUP_DIR/setup.sh
 
-RPI_SETUP_TAG="v5.0.0"
-AVS_DEVICE_SDK_TAG="v1.25.0.2"
+RPI_SETUP_TAG="feature/3510_updates"
```

Comment:
It seems odd to use a feature branch here.  Are you planning to replace it once another pull request completes?